### PR TITLE
Create a warning modal for participants remove

### LINKED ISSUE #2034: Create a warning modal for participants remove
**is a part of https://github.com/taskany-inc/issues/issues/1810**

Notify project owner that participant has actual goals

### PR DESCRIPTION
**Pull request by asabotovich**
## PR includes

- [x] Feature

## Related issues

Resolve #2034 

## QA Instructions, Screenshots, Recordings

https://github.com/taskany-inc/issues/assets/9020793/ea997715-90d4-4de2-8db1-8401b34527b8

### CODE REVIEW

**Comment by asabotovich** [comment]
Yet another attempt

https://github.com/taskany-inc/issues/assets/9020793/dec9dc02-c3f6-48c6-b1fd-d57921b772ad

File: src/components/ProjectParticipants/ProjectParticipants.tsx [hunk 1]
```diff
@@ -24,10 +27,21 @@ const StyledListContianer = styled.div`
...
 }> = ({ project }) => {
+    const [userIdToRemove, setUserIdToRemove] = useState<string>();
     const filterIds = useMemo(() => project.participants.map(({ id }) => id) ?? [], [project]);
 
     const { updateProject } = useProjectResource(project.id);
 
+    const { data: userToRemoveGoals, status } = trpc.project.getActivityGoals.useQuery(
+        {
+            id: project.id,
+            ownerId: userIdToRemove as string,
```
Comment (LamaEats) [inline src/components/ProjectParticipants/ProjectParticipants.tsx:38]: Drop type casting and pass default empty string into `useState` function call

**Review by LamaEats — CHANGES_REQUESTED** [review]

File: src/components/ParticipantDeleteErrorModal/ParticipantDeleteErrorModal.i18n/ru.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "Cannot delete participant now": "Невозможно удалить участника",
+    "The user has actual goals": "На пользователя назначены цели",
+    "Before delete a participant, you must move it`s goals to other person:": "Перед удалением участника из списка назначте его цели другим пользователям:",
```
Comment (tsumo) [inline src/components/ParticipantDeleteErrorModal/ParticipantDeleteErrorModal.i18n/ru.json:4]: Before deleting a participant, you must move their goals to another person

Comment (tsumo) [inline src/components/ParticipantDeleteErrorModal/ParticipantDeleteErrorModal.i18n/ru.json:4]: назначьте

**Review by tsumo — CHANGES_REQUESTED** [review]

File: src/components/ProjectParticipants/ProjectParticipants.tsx (hunk 1 above)
Comment (asabotovich) [reply src/components/ProjectParticipants/ProjectParticipants.tsx:38]: fixed

File: src/components/ParticipantDeleteErrorModal/ParticipantDeleteErrorModal.i18n/ru.json (hunk 2 above)
Comment (asabotovich) [reply src/components/ParticipantDeleteErrorModal/ParticipantDeleteErrorModal.i18n/ru.json:4]: fixed

Comment (asabotovich) [reply src/components/ParticipantDeleteErrorModal/ParticipantDeleteErrorModal.i18n/ru.json:4]: fixed

**Review by LamaEats — APPROVED** [review]

**Review by tsumo — APPROVED** [review]

**Pull request merged**